packages/components/src/custom-select-control/index.tsx

```diff
@@ -48,8 +64,13 @@ function CustomSelectControl( props: CustomSelectProps ) {
 		...restProps
 	} = useDeprecatedProps( props );
 
+	const descriptionId = useInstanceId(
+		CustomSelectControl,
+		'custom-select-control__description'
+	);
+
 	// Forward props + store from v2 implementation
-	const store = Ariakit.useSelectStore( {
+	const store = Ariakit.useSelectStore< string >( {
```

Comment:
The Ariakit store allows `string | string[]` because it can handle multiple selection. Since this v1 never supported multiple selection, let's narrow the type to just `string`. (This has the effect of narrowing the type of `store.getState().value` to string.)